src/Handler/DateHandler.php

```diff
@@ -69,9 +74,17 @@ public static function getSubscribingMethods()
         return $methods;
     }
 
-    public function __construct(string $defaultFormat = \DateTime::ATOM, string $defaultTimezone = 'UTC', bool $xmlCData = true)
-    {
+    /**
+     * @param array<string> $defaultDeserializationFormats
+     */
+    public function __construct(
+        string $defaultFormat = \DateTime::ATOM,
+        string $defaultTimezone = 'UTC',
+        bool $xmlCData = true,
+        array $defaultDeserializationFormats = [\DateTime::ATOM]
+    ) {
         $this->defaultFormat = $defaultFormat;
+        $this->defaultDeserializationFormats = $defaultDeserializationFormats;
```

Comment:
If I understand correctly in the past `$defaultFormat` was working for both - serialisation & deserialisation. With the changes that you proposed I can see small BC break - when somebody was using $defaultFormat with different format after updating the package, they will get `DateTime::ATOM` as a deserialisation format what might be small BC. I guess something like might help to keep it consistent with old behaviour when no `defaultDeserializationFormats` are passed. 
```suggestion
        bool $xmlCData = true,
        array $defaultDeserializationFormats = []
    ) {
        $this->defaultFormat = $defaultFormat;
        $this->defaultDeserializationFormats = $defaultDeserializationFormats === [] ?  [$defaultFormat] : $defaultDeserializationFormats;
```

Reply:
Ou, you are very correct. In my case, it would be BC. Thanks for providing me with a solution for this.
I was thinking about renaming private property `defaultFormat` to `defaultSerializationFormat`, what do you think?

Reply:
Renaming sounds like a good solution 👍

Reply:
OK, done